pages/docs/support/rest-api.mdx

```diff
@@ -0,0 +1,215 @@
+  -H 'accept: application/json'
+```
+
+with your instance's URL.
+
+## Using the Grouparoo API
+
+### Generating an API Key
+
+To get started using Grouparoo's REST API, you'll need to generate an API key. To do this, first make sure you are signed into an account with "write" access for API Keys. Then:
+
+#### UI Method
+
+1. Go to Platform > API Keys or navigate to `/apiKeys` and select "Add API Key".
+
+2. Give your API Key a name. An API Key will be generated and you will be prompted to set permissions for this API.
+
+<ImageInBrowserFrame
+  centered
+  source="Generate an API Key"
+  alt="Generate an API Key"
+  src="/images/docs/support/rest-api/apikeyconfig.png"
+  width={820}
+  height={502}
+/>
+
+To learn more about what each of the topics can do in the API, you can access the Swagger Docs for the Grouparoo API by navigating to `/swagger` in the UI of your Grouparoo Instance or going to Platform > API Documentation.
+
+#### Code Config Method
+
+You can also generate an API key object with the [`grouparoo generate` command]. To do this, run
+
+```
+grouparoo generate apiKey {{yourAPIKeyId}}
+```
+
+This command will generate a config file in a new directory `/config/apiKeys/{{yourAPIKeyId}}.js` that looks like this:
+
+```js
+exports.default = async function buildConfig() {
+  return [
+    {
+      class: "apiKey",
+      id: "{{yourAPIKeyId}}",
+      name: "{{yourAPIKeyId}}",
+      apiKey: "API KEY HERE" // you can hard-code the API key you want to use, or source it from `process.env.PRODUCTION_API_KEY`
+      options: {
+        permissionAllRead: false, // Does this API Key have read permission to all topics?
+        permissionAllWrite: false, // Does this API Key have write permission to all topics?
+      },
+    },
+  ];
+};
+```
+
+Grouparoo will auto-generate an API key for you, but it is also possible to set it explicitly in the config file or as an env variable `PRODUCTION_API_KEY`.
+
+From here, you can update the name and permissions.
+
+### Accessing & Testing the API
+
+As Grouparoo can be self-hosted, every deployment of Grouparoo provides its own API documentation.
+The Swagger docs for your API can be found by navigating to `/swagger` in the Grouparoo UI.
+
+<ImageInBrowserFrame
+  centered
+  source="Generate an API Key"
+  alt="Generate an API Key"
+  src="/images/docs/support/rest-api/swaggerdocs.png"
+  width={820}
+  height={530}
+/>
+
+The Grouparoo API can be authorized in 2 distinct ways: as a web user (with a cookie and CSRF token), or via an API Key. By default, when visiting the `/swagger` page, you will be authenticated using your existing cookie. You can switch to using an API key by clicking "Authorize" and entering your API Key (if pasting in your API Key, be careful about whitespace).
+
+<ImageInBrowserFrame
+  centered
+  source="Generate an API Key"
+  alt="Generate an API Key"
+  src="/images/docs/support/rest-api/auths.png"
+  width={820}
+  height={521}
+/>
+
```

Comment:
I think an bridge sentance is needed here - we are jumping from the swagger UI to the command line.  How about:

```suggestion

You can use the Swagger UI to try out all the APIs available to your Grouparoo instance.  The Grouparoo API is  available to use via any HTTP API Client.  In the following examples, we'll use the ubiquitous [cURL](https://curl.se/) library. 
```